File: Java/Homework02/Homework02_02/Homework02_02.java
/*
 * File name : Homework02_02.java
 * Author : Kim Yeong Ho
 * Date : July 11, 2023
 * Major feature : 
 * - 숫자를 입력받아 3~5는 "봄", 6~8은 "여름", 9~11은 "가을",
 * - 12~2는 "겨울"로, 그 이외의 숫자를 입력한 경우에는 "잘못입력"을 출력하는 프로그램을 작성하시오
 * - 달을 입력하시오(1~12)>>9
 * - 가을
 */
package Homework02_02;

import java.util.Scanner;

public class Homework02_02 {
	public static void main(String[] args) {
		final String[] seasons = {"겨울", "봄", "여름", "가을", "겨울"};
		int month;
		String result;
		Scanner scanner = new Scanner(System.in);

		System.out.print("달을 입력하시오(1~12) : ");
		month = scanner.nextInt();

		result = (0 < month && month < 13) ? seasons[month / 3] : "잘못입력";
		System.out.println(result);

		scanner.close();
	}
}

File: C/Homework02_02/Homework02_02.c
/*
* File name : Homework02_02.c
* Author : Kim Yeong Ho
* Date : July 24, 2023
* Major feature :
* - 정수 1개를 입력(a)하고, 4를 몇 번 곱할 것인지 입력(p)하세요
* - 입력된 정수에 4를 입력한 만큼 곱한 값 10진수로 우선 출력하세요
*  - a x 4^p 계산 결과가 되어야 합니다
*   - 입력 a에 4^p를 곱하는 곱셈 연산은 반드시 비트연산자를 이용하세요
*   - 나온 값을 16진수를 출력할 때 앞에 0x를 붙여주세요
*     - ex) 0x0, 0x1CFF, 0x22 등등 자리 수는 무관합니다.
*/
#include <stdio.h>

int main(void)
{
	int input, times, result;

	printf("정수를 입력하시오 : ");
	scanf("%d", &input);

	printf("4을(를) 곱하고 싶은 횟수 : ");
	scanf("%d", &times);

	result = input * (4 << (times - 1) * 2);
	printf("%d x 4^%d = %d\n", input, times, result);
	printf("16진수로 변환한 값 : 0x%X", result);
	return 0;
}